include/pmacc/PMaccConfig.cmake

```diff
@@ -314,6 +318,7 @@ endif()
 if( ("${PMACC_CUDA_COMPILER}" STREQUAL "nvcc") AND
```

Comment:
@BeyondEspresso here is your bug on Taurus (TUD): `PMACC_CUDA_COMPILER` is an empty string which makes the work-around not active as it should.

will investigate why.

Reply:
The work-around was added in #2076 but the controlling var vanished during the marriage with alpaka in #2178

Reply:
We can just use `ALPAKA_CUDA_COMPILER` now.